.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
```

Comment:
What's all this about?

Reply:
It came by default with cookie cutter template which I used to create/update this python package. It depends on if we would like to have git pre-commit hooks with formatting or not. Also basic checks are included in this step. In theory it should help to keep good-looking python code in terms of style before committing it to the repo.